packages/@coorpacademy-components/src/organism/filter-checkbox-and-search/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {useCallback, useMemo, useState} from 'react';
+import {filter, pipe, size, toString} from 'lodash/fp';
+import Title from '../../atom/title';
+import Tag from '../../atom/tag';
+import ButtonLink from '../../atom/button-link';
+import SearchForm from '../../molecule/search-form';
+
+import Provider, {GetTranslateFromContext} from '../../atom/provider';
+import {WebContextValues} from '../../atom/provider/web-context';
+import CheckboxWithTitle from '../../atom/checkbox-with-title';
+import {COLORS} from '../../variables/colors';
+import style from './style.css';
+import propTypes, {FilterCheckboxAndSearchProps} from './props-types';
+
+const CLEAR_BUTTON_STYLE = {fontWeight: 'normal', padding: 0};
+const SHOW_BUTTON_STYLE = {
+  fontSize: '14px',
+  fontWeight: '600',
+  fontFamily: 'Gilroy',
+  marginTop: '16px'
+};
+const CHECKBOX_TITLE_STYLE = {
+  marginRight: '16px',
+  fontSize: '14px',
+  fontFamily: 'Gilroy',
+  fontWeight: 'normal'
+};
+const INITIAL_VISIBLE_OPTIONS = 5;
+
+const FilterCkeckboxAndSearch = (
+  props: FilterCheckboxAndSearchProps,
+  context: WebContextValues
+) => {
+  const {title, onSearchProps, onClear, options} = props;
+  const translate = GetTranslateFromContext(context);
+  const [showMore, setShowMore] = useState(false);
+  const selectedFiltersCount = pipe(filter({selected: true}), size)(options);
+  const hasSelectedFilters = selectedFiltersCount > 0;
+
+  const handleShowMore = useCallback(() => {
+    setShowMore(!showMore);
+  }, [showMore]);
+
+  const visibleOptions = useMemo(() => {
+    return showMore ? options : options.slice(0, INITIAL_VISIBLE_OPTIONS);
+  }, [options, showMore]);
+
+  return (
+    <div data-testid="filter-checkbox-and-search-container">
+      <div className={style.header} data-testid="filter-checkbox-and-search-header">
+        <div
+          className={style.titleAndTagWrapper}
+          data-testid="filter-checkbox-and-searh-title-and-tag-wrapper"
+        >
+          <Title title={title} />
+          {hasSelectedFilters ? (
+            <Tag label={toString(selectedFiltersCount)} type="info" size="S" />
+          ) : null}
+        </div>
+        {hasSelectedFilters ? (
+          <div className={style.clearButtonContainer}>
+            <ButtonLink
+              label={'Clear'}
```

Comment:
```suggestion
              label={translate('filter_clear')}
```
ajoute une locale dans locales/en/global.json